test/mlir/onnx/onnx_constprop.mlir

```diff
@@ -195,3 +195,34 @@ func @test_neg_3(%arg0: tensor<3x2xi32>) -> tensor<3x2xi32> {
   // CHECK-NEXT: [[ADD1:%.+]] = "onnx.Add"(%arg0, [[CONST1]]) : (tensor<3x2xi32>, tensor<3x2xi32>) -> tensor<3x2xi32>
 }
 
+//===----------------------------------------------------------------------===//
+/// Transpose tests.
```

Comment:
Missing the lower delimiter for comment section:
i.e., //===----------------------------------------------------------------------===//